moveit_setup_assistant/templates/moveit_config_pkg_template/launch/run_benchmark_ompl.launch

```diff
@@ -16,6 +16,7 @@
   <!-- Start Benchmark Executable -->
   <node name="$(anon moveit_benchmark)" pkg="moveit_ros_benchmarks" type="moveit_run_benchmark" args="$(arg cfg) --benchmark-planners" respawn="false" output="screen">
     <rosparam command="load" file="$(find [GENERATED_PACKAGE_NAME])/config/kinematics.yaml"/>
+    [KINEMATICS_PARAMETERS_FILE_NAMES_BLOCK]
```

Comment:
See above?

Reply:
Do you have a working setup were you could quickly test this? I have never used the benchmarking so far

Reply:
Me neither.
I noticed `planning_context.launch` is included in this file too, so the parameters (in the `move_group` namespace) are set.
I'll forward the question to @henningkayser .

Reply:
I tested the benchmark and loading the parameters here is indeed superfluous, removed it